apps/cli/src/compose/coprocessor/docker-compose-carize.yaml

```diff
@@ -0,0 +1,7 @@
+services:
+    carize:
+        image: ghcr.io/zippiehq/cartesi-carize:latest
```

Comment:
we always use tagged versions, for stability

Reply:
Can the car generation be implemented in typescript and remove the need for an outside docker dependency?
I played a lot with CAR in typescript before, and it's usually straightforward.
The tradeoff is adding another npm dependency vs downloading another docker dependency.

https://www.npmjs.com/package/ipfs-car